community/feature-status.md

```diff
@@ -0,0 +1,152 @@
+# Feature Status
+
+This document describes the different statuses a given feature can
+have in Argo projects.
+
+## Overview
+
+Argo umbrella projects are maintained by the open-source community.
+This means that features are provided by different companies with
+different use-cases. While certain features are battle tested by
+corporations with large infrastructure, this is not always the case.
+Feature status is a classification that can be used by Argo projects
+to clarify and set expectations about a given functionality. However,
+not every feature will have to be associated with a given status.
+Maintainers will use this classification as a framework and gradually
+associate features with the proper current status. When available, the
+current status will be informed in the documentation of that specific
+feature.
+
+## Definition
+
+Argo projects adopt the standard software life cycle terminology to
+define the different feature statuses:
+
+- _Alpha_
+- _Beta_
+- _Stable_
```

Comment:
Wondering if we should call it "GA" instead of "Stable".

Reply:
Good point. I think I have a slight preference for `Stable` to avoid the acronym but I am not strong opinionated about that.

Reply:
+1 for `Stable`. It's also the rhetoric the doc versioning uses.